# datasets
Simulated and biological datasets from publications from Tandy Warnow's lab, mostly from her years at UTexas.

<table >

<tr >

<th  >

<span >Authors and/or software name</span>

</th>

<th  >

<span >Publication title, journal (year)</span>

</th>

<th  >

<span >Paper DOI</span>

</th>

<th  >

<span >Old Address</span>

</th>

<th  >

<span >New Address</span>

</th>

</tr>

<tbody>
<tr >

<td  >

<span >Mirarab et al., ASTRAL-I</span>

</td>

<td  >

<span >ASTRAL: Genome-Scale Coalescent-Based Species Tree Estimation, Bioinformatics (2014)</span>

</td>

<td  >

<span>[10.1093/bioinformatics/btu462](https://www.google.com/url?q=http://bioinformatics.oxfordjournals.org/cgi/content/long/30/17/i541&sa=D&ust=1508540976065000&usg=AFQjCNHGymI3SLe4jIBTh2Od4lIk1oiJ4w)</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/~phylo/datasets/astral/](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/astral/&sa=D&ust=1508540976065000&usg=AFQjCNGON0zQw-oJzaKwU5hS7p-Hn_gh7g)</span><span > </span>

<span ></span>

<span >[http://www.cs.utexas.edu/~phylo/software/astral/](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/astral/&sa=D&ust=1508540976066000&usg=AFQjCNEmuLN5Wmn0YudRWQESXbeKp9Ia7g)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/astral](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/astral&sa=D&ust=1508540976066000&usg=AFQjCNEhlJfbGdQytoY7T8exp74xqihmeg)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Mirarab and Warnow, ASTRAL-II</span>

</td>

<td  >

<span >ASTRAL-II: coalescent-based species tree estimation with many hundreds of taxa and thousands of genes, Bioinformatics (2015)</span>

</td>

<td  >

<span >10.1093/bioinformatics/btv234</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/~phylo/datasets/astral2/](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/astral2/&sa=D&ust=1508540976067000&usg=AFQjCNFPRmha9biIQ358Ww2qMsYViq1qFQ)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/astral-ii](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/astral-ii&sa=D&ust=1508540976067000&usg=AFQjCNHuqXe0hOobEEKPht8Rqw5r9pHwPA)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Mirarab et al., Statistical binning</span>

</td>

<td  >

<span >Statistical binning enables an accurate coalescent-based estimation of the avian tree, Science (2014)</span>

</td>

<td  >

<span >10.1126/science.1250463</span>

</td>

<td  >

<span >[www.cs.utexas.edu/~phylo/datasets/binning](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/binning&sa=D&ust=1508540976068000&usg=AFQjCNES0j1mPcZOanqNqd209NU7-hIoCA)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/binning](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/binning&sa=D&ust=1508540976069000&usg=AFQjCNFOWe-Qsx4W8YzcoDGKc16soEQJBg)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Mirarab et al., Response to comment on binning</span>

</td>

<td  >

<span >Response to Comment on ‘Statistical Binning Enables an Accurate Coalescent-Based Estimation of the Avian Tree.’ Science (2015)</span>

</td>

<td  >

<span >10.1126/science.aaa7719</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/~phylo/datasets/binning-response](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/binning-response/&sa=D&ust=1508540976070000&usg=AFQjCNE5EH98ovHaNsbDxIPh2qCmdaqtPA)</span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/binning](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/binning&sa=D&ust=1508540976070000&usg=AFQjCNGFznBLSZZUXBqGyv29Sc-U4iwdXw)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Bayzid et al., Weighted statistical binning</span>

</td>

<td  >

<span >Weighted Statistical Binning: Enabling Statistically Consistent Genome-Scale Phylogenetic Analyses, PLOS One (2015)</span>

</td>

<td  >

<span >10.1371/journal.pone.0129183</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/~phylo/datasets/weighted-binning](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/weighted-binning/&sa=D&ust=1508540976071000&usg=AFQjCNGC5NsZ4aUjr0J7-DB-zi9vaJPHkw)</span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/binning](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/binning&sa=D&ust=1508540976072000&usg=AFQjCNEzdB8x0eyYJi7CJwa7ipaMUcn_qw)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Bayzid and Warnow,
Naive binning</span>

</td>

<td  >

<span >Naive binning improves phylogenomic analyses, Bioinformatics (2013)</span>

</td>

<td  >

<span >10.1093/bioinformatics/btt394</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/users/phylo/datasets/ILS/](https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/datasets/ILS/&sa=D&ust=1508540976073000&usg=AFQjCNFjaf57Zxli_0hrno71Pl1IBAyhiw)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/ils-small](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/ils-small&sa=D&ust=1508540976073000&usg=AFQjCNHdPUZmQIwNm1EDWWLwtxNoiWqMqQ)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Zimmermann, Mirarab, and Warnow, BBCA</span>

</td>

<td  >

<span >BBCA: Improving the scalability of *BEAST using random binning, BMC Genomics (2014)</span>

</td>

<td  >

<span >[10.1186/1471-2164-15-S6-S11](https://www.google.com/url?q=http://www.biomedcentral.com/1471-2164/15/S6/S11&sa=D&ust=1508540976074000&usg=AFQjCNFWi90IuuWHmKB74vdbEpgyI6bGcQ)</span>

<span ></span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/~phylo/datasets/bbca/](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/bbca/&sa=D&ust=1508540976074000&usg=AFQjCNEPVwCTuaRxSJdDDWclYHnsLxmVBg)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/binning](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/binning&sa=D&ust=1508540976075000&usg=AFQjCNFrbo3RoTrYEmz0GUyZNhQwGYv0Lw)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Mirarab et al., PASTA</span>

</td>

<td  >

<span >PASTA: Ultra-Large Multiple Sequence Alignment for Nucleotide and Amino-Acid Sequences, Journal of Computational Biology (2014)</span>

</td>

<td  >

<span >10.1089/cmb.2014.0156</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/~phylo/software/pasta/](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/pasta/&sa=D&ust=1508540976076000&usg=AFQjCNHCRzOyaypUihgsYujfSdzG8DJnGQ)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/pastaupp](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/pastaupp&sa=D&ust=1508540976076000&usg=AFQjCNECJeApMK076AuB_kOT7g-iMmAehw)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Mirarab et al., SEPP</span>

</td>

<td  >

<span >SEPP: SATé-Enabled Phylogenetic Placement, Pacific Symposium on Biocomputing (2012)</span>

</td>

<td  >

<span >10.1142/9789814366496_0024</span><span > </span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/~phylo/software/sepp/submission/](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/sepp/submission/&sa=D&ust=1508540976077000&usg=AFQjCNHpE4FU1iWvopc3BoM0cm9Xc8chKg)</span><span > </span>

<span >[http://www.cs.utexas.edu/~phylo/software/sepp/](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/sepp/&sa=D&ust=1508540976078000&usg=AFQjCNEle58c_46cCon2Rf8Y0QqmPo8CzQ)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/sepp](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/sepp&sa=D&ust=1508540976078000&usg=AFQjCNHtasOzJIgJHpBKvYoLZtbspn-4dA)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Nguyen et al., UPP</span>

</td>

<td  >

<span >Ultra-large alignments using phylogeny aware profiles, Genome Biology (2015)</span>

</td>

<td  >

<span >10.1186/s13059-015-0688-z</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/users/phylo/software/upp/](https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/upp/&sa=D&ust=1508540976079000&usg=AFQjCNHKIAoCyoSw35wGCjpqRL3gWldFrg)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/pastaupp](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/pastaupp&sa=D&ust=1508540976080000&usg=AFQjCNF8VeUi_OmeMF6m2_9HT0U8VY0Dxg)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Nguyen et al., TIPP</span>

</td>

<td  >

<span >TIPP:Taxonomic Identification and Phylogenetic Profiling, Bioinformatics (2014)</span>

</td>

<td  >

<span >10.1093/bioinformatics/btu721</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/users/phylo/software/sepp/tipp-submission/](https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/sepp/tipp-submission/&sa=D&ust=1508540976081000&usg=AFQjCNFIfldduXE2QEWyREq7u_-tbfDtug)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/tipp](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/tipp&sa=D&ust=1508540976081000&usg=AFQjCNGKBx8YsLXmdqASnlqZXUdXkmJ4qg)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Mirarab and Warnow, FastSP</span>

</td>

<td  >

<span >FastSP: Linear Time Calculation of Alignment Accuracy, Bioinformatics (2011)</span>

</td>

<td  >

<span >[10.1093/bioinformatics/btr553](https://www.google.com/url?q=http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btr553&sa=D&ust=1508540976082000&usg=AFQjCNEplLhSI3UYCNRVzvu1Beyb5KTaKg)</span>

<span ></span>

</td>

<td  >

<span >[www.cs.utexas.edu/~phylo/software/fastsp](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/fastsp&sa=D&ust=1508540976083000&usg=AFQjCNE-SUJAwuS7Tu1k4zVz7pk4rgoKbg)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/fastsp](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/fastsp&sa=D&ust=1508540976083000&usg=AFQjCNHvpzpXRAgg4wgfjsOBLc2qQ-B35w)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Liu et al., SATe-I</span>

</td>

<td  >

<span >Rapid and Accurate Large-Scale Coestimation of Sequence Alignments and Phylogenetic Trees, Science (2009)</span>

</td>

<td  >

<span >10.1126/science.1171243</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/~phylo/sate/public/sate_journal.html](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/sate/public/sate_journal.html&sa=D&ust=1508540976084000&usg=AFQjCNFbMgFCE4ymM4S8lPtI0kx105KU3w)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/sate-i](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/sate-i&sa=D&ust=1508540976084000&usg=AFQjCNF-UG0t8TV6yMnp_dhQa23IgTZOtQ)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Swenson et al.</span>

<span >SuperFine</span>

</td>

<td  >

<span >SuperFine: fast and accurate supertree estimation.</span>

</td>

<td  >

<span >10.1093/sysbio/syr092</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/users/phylo/software/superfine/](https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/superfine/&sa=D&ust=1508540976085000&usg=AFQjCNEZQPr_T5GjvFZv0zMxS8GdbWB5sg)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine&sa=D&ust=1508540976086000&usg=AFQjCNEYwCXWzbkSeuMBzjDfzU_hlQeOqA)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Nelesen et al., DACTAL</span>

</td>

<td  >

<span >DACTAL: divide-and-conquer trees (almost) without alignments, Bioinformatics (2012)</span>

</td>

<td  >

<span >10.1093/bioinformatics/bts218</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/users/phylo/software/dactal/](https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/dactal/&sa=D&ust=1508540976087000&usg=AFQjCNGj_mL0xsZ_ph72k9xIwUt35I-rYQ)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine&sa=D&ust=1508540976087000&usg=AFQjCNGwfJws2afXHRPoysbc3eq-krzjzQ)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Swenson et al., SMIDGEN</span>

</td>

<td  >

<span >A simulation study comparing supertree and combined analysis methods using SMIDGen, Algorithms for Molecular Biology (2010)</span>

</td>

<td  >

<span >https://doi.org/10.1186/1748-7188-5-8</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/users/mswenson/pubs/](https://www.google.com/url?q=http://www.cs.utexas.edu/users/mswenson/pubs/&sa=D&ust=1508540976088000&usg=AFQjCNEFjQjXGsDbekiQ66guTHLDsIUaGA)</span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine&sa=D&ust=1508540976089000&usg=AFQjCNFsi0fcmne6eMSVNhhPM90YZhHWzw)</span>

</td>

</tr>

<tr >

<td  >

<span >Liu and Warnow (BeeTLe)</span>

</td>

<td  >

<span >Treelength Optimization for Phylogeny Estimation, PLOS One (2012)</span>

</td>

<td  >

<span >[10.1371/journal.pone.0033104](doi.org/10.1371/journal.pone.0033104)</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/users/phylo/datasets/treelength/](https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/datasets/treelength/&sa=D&ust=1508540976091000&usg=AFQjCNGG6UgjAw3sT8nU_tFOS4VKhWSiLA)</span><span > </span>

<span >(data)</span>

<span ></span>

<span >[http://www.cs.utexas.edu/users/phylo/software/beetle/beetle-may-17-2013.tar.bz2](https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/beetle/beetle-may-17-2013.tar.bz2&sa=D&ust=1508540976092000&usg=AFQjCNEN8YTgI1iDrotTEtXOfsj7pJ9VAQ)</span><span > </span>

<span >(software)</span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine&sa=D&ust=1508540976093000&usg=AFQjCNGgf2U5Cav7n-BhEBawWd6MmJeCaA)</span>

<span ></span>

</td>

</tr>

<tr >

<td  >

<span >Liu et al.</span>

</td>

<td  >

<span >RAxML and FastTree: Comparing Two Methods for Large-Scale Maximum Likelihood Phylogeny Estimation, PLOS One (2012)</span>

</td>

<td  >

<span >10.1371/journal.pone.0027731</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/~phylo/sate/public/sate_journal.html](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/sate/public/sate_journal.html&sa=D&ust=1508540976094000&usg=AFQjCNExLwG0GLMtA7vujuKCkpPDT007yg)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/sate-i](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/sate-i&sa=D&ust=1508540976095000&usg=AFQjCNE5oVN5t-byPEO83HChz7sNd1l9GQ)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Yang and Warnow</span>

</td>

<td  >

<span >Fast and accurate methods for phylogenomic analyses, BMC Bioinformatics (2011)</span>

</td>

<td  >

<span >[10.1093/bioinformatics/btt394](https://www.google.com/url?q=https://doi.org/10.1093/bioinformatics/btt394&sa=D&ust=1508540976096000&usg=AFQjCNHYhJ3bhNh67BRakSrLs808y4yQrg)</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/users/phylo/datasets/ILS/](https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/datasets/ILS/&sa=D&ust=1508540976096000&usg=AFQjCNEOq1cS5LlGko_oBgjcOmTre4EV4A)</span><span > </span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/ils-small](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/ils-small&sa=D&ust=1508540976097000&usg=AFQjCNFE7m5t4vQ7ACKCxphVYnWuwh5Mcw)</span><span > </span>

</td>

</tr>

<tr >

<td  >

<span >Bayzid, Hunt, and Warnow</span>


</td>

<td  >

<span >Disk covering methods improve phylogenomc analyses, BMC Genomics (2014)</span>

</td>

<td  >

<span >[10.1186/1471-2164-15-S6-S7](https://www.google.com/url?q=https://doi.org/10.1186/1471-2164-15-S6-S7&sa=D&ust=1508540976098000&usg=AFQjCNF7hKuhu38RUNj4DrJRpoQrSYTdfA)</span>

</td>

<td  >

<span >[http://www.cs.utexas.edu/users/phylo/software/dcm-protocol/](https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/dcm-protocol/&sa=D&ust=1508540976099000&usg=AFQjCNEDtribVivnacPPL2uPaevYcvi5hA)</span><span > </span><span >(scripts)</span>

<span ></span>

<span >[http://www.cs.utexas.edu/~phylo/datasets/astral/](https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/astral/&sa=D&ust=1508540976099000&usg=AFQjCNFOBb8qwYg7WCjdyV5RLhFCINB1tQ)</span><span > </span><span >(datasets)</span>

</td>

<td  >

<span >[https://sites.google.com/eng.ucsd.edu/datasets/ils-small](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/ils-small&sa=D&ust=1508540976100000&usg=AFQjCNFtBFhmwMnYall3wa66Z_ALrne0Sw)</span><span > 
(scripts)</span>

<span ></span>

<span >[https://sites.google.com/eng.ucsd.edu/datasets/astral](https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/astral&sa=D&ust=1508540976100000&usg=AFQjCNET0QFRWZ8Dgrs1Pzy_VgR6yR8Zgw)</span><span > </span><span > (datasets)</span>

</td>

</tr>

</tbody>

</table>
