# datasets
Simulated and biological datasets from publications from Tandy Warnow's lab, mostly from her years at UTexas. 



<table class="c33"><tbody><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c0">Authors and/or software name</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c0">Publication title, journal (year)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c0">Paper DOI</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c0">Old Address</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c0">New Address</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Mirarab et al., ASTRAL-I</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">ASTRAL: Genome-Scale Coalescent-Based Species Tree Estimation, Bioinformatics (2014)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c22 c35"><a class="c10" href="https://www.google.com/url?q=http://bioinformatics.oxfordjournals.org/cgi/content/long/30/17/i541&amp;sa=D&amp;ust=1508540976065000&amp;usg=AFQjCNHGymI3SLe4jIBTh2Od4lIk1oiJ4w">10.1093/bioinformatics/btu462</a></span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/astral/&amp;sa=D&amp;ust=1508540976065000&amp;usg=AFQjCNGON0zQw-oJzaKwU5hS7p-Hn_gh7g">http://www.cs.utexas.edu/~phylo/datasets/astral/</a></span><span class="c3">&nbsp;</span></p><p class="c2 c7"><span class="c3"></span></p><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/astral/&amp;sa=D&amp;ust=1508540976066000&amp;usg=AFQjCNEmuLN5Wmn0YudRWQESXbeKp9Ia7g">http://www.cs.utexas.edu/~phylo/software/astral/</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/astral&amp;sa=D&amp;ust=1508540976066000&amp;usg=AFQjCNEhlJfbGdQytoY7T8exp74xqihmeg">https://sites.google.com/eng.ucsd.edu/datasets/astral</a></span><span class="c9">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Mirarab and Warnow, ASTRAL-II</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">ASTRAL-II: coalescent-based species tree estimation with many hundreds of taxa and thousands of genes, Bioinformatics (2015)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3">10.1093/bioinformatics/btv234</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/astral2/&amp;sa=D&amp;ust=1508540976067000&amp;usg=AFQjCNFPRmha9biIQ358Ww2qMsYViq1qFQ">http://www.cs.utexas.edu/~phylo/datasets/astral2/</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/astral-ii&amp;sa=D&amp;ust=1508540976067000&amp;usg=AFQjCNHuqXe0hOobEEKPht8Rqw5r9pHwPA">https://sites.google.com/eng.ucsd.edu/datasets/astral-ii</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Mirarab et al., Statistical binning</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">Statistical binning enables an accurate coalescent-based estimation of the avian tree, Science (2014)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c22 c29 c12">10.1126/science.1250463</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/binning&amp;sa=D&amp;ust=1508540976068000&amp;usg=AFQjCNES0j1mPcZOanqNqd209NU7-hIoCA">www.cs.utexas.edu/~phylo/datasets/binning</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/binning&amp;sa=D&amp;ust=1508540976069000&amp;usg=AFQjCNFOWe-Qsx4W8YzcoDGKc16soEQJBg">https://sites.google.com/eng.ucsd.edu/datasets/binning</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Mirarab et al., Response to comment on binning</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">Response to Comment on &lsquo;Statistical Binning Enables an Accurate Coalescent-Based Estimation of the Avian Tree.&rsquo; Science (2015)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c19 c20 c12">10.1126/science.aaa7719</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/binning-response/&amp;sa=D&amp;ust=1508540976070000&amp;usg=AFQjCNE5EH98ovHaNsbDxIPh2qCmdaqtPA">http://www.cs.utexas.edu/~phylo/datasets/binning-response</a></span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/binning&amp;sa=D&amp;ust=1508540976070000&amp;usg=AFQjCNGFznBLSZZUXBqGyv29Sc-U4iwdXw">https://sites.google.com/eng.ucsd.edu/datasets/binning</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Bayzid et al., Weighted statistical binning</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">Weighted Statistical Binning: Enabling Statistically Consistent Genome-Scale Phylogenetic Analyses, PLOS One (2015)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3">10.1371/journal.pone.0129183</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/weighted-binning/&amp;sa=D&amp;ust=1508540976071000&amp;usg=AFQjCNGC5NsZ4aUjr0J7-DB-zi9vaJPHkw">http://www.cs.utexas.edu/~phylo/datasets/weighted-binning</a></span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/binning&amp;sa=D&amp;ust=1508540976072000&amp;usg=AFQjCNEzdB8x0eyYJi7CJwa7ipaMUcn_qw">https://sites.google.com/eng.ucsd.edu/datasets/binning</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Bayzid and Warnow, <br>Naive binning</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c22">Naive binning improves phylogenomic analyses, Bioinformatics (2013)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c22">10.1093/bioinformatics/btt394 </span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c13"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/datasets/ILS/&amp;sa=D&amp;ust=1508540976073000&amp;usg=AFQjCNFjaf57Zxli_0hrno71Pl1IBAyhiw">http://www.cs.utexas.edu/users/phylo/datasets/ILS/</a></span><span class="c9">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/ils-small&amp;sa=D&amp;ust=1508540976073000&amp;usg=AFQjCNHdPUZmQIwNm1EDWWLwtxNoiWqMqQ">https://sites.google.com/eng.ucsd.edu/datasets/ils-small</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Zimmermann, Mirarab, and Warnow, BBCA</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">BBCA: Improving the scalability of *BEAST using random binning, BMC Genomics (2014)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c19 c9"><a class="c10" href="https://www.google.com/url?q=http://www.biomedcentral.com/1471-2164/15/S6/S11&amp;sa=D&amp;ust=1508540976074000&amp;usg=AFQjCNFWi90IuuWHmKB74vdbEpgyI6bGcQ">10.1186/1471-2164-15-S6-S11</a></span></p><p class="c2 c7"><span class="c3"></span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/bbca/&amp;sa=D&amp;ust=1508540976074000&amp;usg=AFQjCNEPVwCTuaRxSJdDDWclYHnsLxmVBg">http://www.cs.utexas.edu/~phylo/datasets/bbca/</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/binning&amp;sa=D&amp;ust=1508540976075000&amp;usg=AFQjCNFrbo3RoTrYEmz0GUyZNhQwGYv0Lw">https://sites.google.com/eng.ucsd.edu/datasets/binning</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Mirarab et al., PASTA</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">PASTA: Ultra-Large Multiple Sequence Alignment for Nucleotide and Amino-Acid Sequences, Journal of Computational Biology (2014)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3">10.1089/cmb.2014.0156</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/pasta/&amp;sa=D&amp;ust=1508540976076000&amp;usg=AFQjCNHCRzOyaypUihgsYujfSdzG8DJnGQ">http://www.cs.utexas.edu/~phylo/software/pasta/</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/pastaupp&amp;sa=D&amp;ust=1508540976076000&amp;usg=AFQjCNECJeApMK076AuB_kOT7g-iMmAehw">https://sites.google.com/eng.ucsd.edu/datasets/pastaupp</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Mirarab et al., SEPP</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">SEPP: SAT&eacute;-Enabled Phylogenetic Placement, Pacific Symposium on Biocomputing (2012)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.ncbi.nlm.nih.gov/pubmed/22174280&amp;sa=D&amp;ust=1508540976077000&amp;usg=AFQjCNEoD9I7aeUshg0Ov08S87f91De81g">http://www.ncbi.nlm.nih.gov/pubmed/22174280</a></span><span class="c3">&nbsp;</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/sepp/submission/&amp;sa=D&amp;ust=1508540976077000&amp;usg=AFQjCNHpE4FU1iWvopc3BoM0cm9Xc8chKg">http://www.cs.utexas.edu/~phylo/software/sepp/submission/</a></span><span class="c3">&nbsp;</span></p><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/sepp/&amp;sa=D&amp;ust=1508540976078000&amp;usg=AFQjCNEle58c_46cCon2Rf8Y0QqmPo8CzQ">http://www.cs.utexas.edu/~phylo/software/sepp/</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/sepp&amp;sa=D&amp;ust=1508540976078000&amp;usg=AFQjCNHtasOzJIgJHpBKvYoLZtbspn-4dA">https://sites.google.com/eng.ucsd.edu/datasets/sepp</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Nguyen et al., UPP</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">Ultra-large alignments using phylogeny aware profiles, Genome Biology (2015)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3">10.1186/s13059-015-0688-z </span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/upp/&amp;sa=D&amp;ust=1508540976079000&amp;usg=AFQjCNHKIAoCyoSw35wGCjpqRL3gWldFrg">http://www.cs.utexas.edu/users/phylo/software/upp/</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/pastaupp&amp;sa=D&amp;ust=1508540976080000&amp;usg=AFQjCNF8VeUi_OmeMF6m2_9HT0U8VY0Dxg">https://sites.google.com/eng.ucsd.edu/datasets/pastaupp</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Nguyen et al., TIPP</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">TIPP:Taxonomic Identification and Phylogenetic Profiling, Bioinformatics (2014)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3">10.1093/bioinformatics/btu721</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/sepp/tipp-submission/&amp;sa=D&amp;ust=1508540976081000&amp;usg=AFQjCNFIfldduXE2QEWyREq7u_-tbfDtug">http://www.cs.utexas.edu/users/phylo/software/sepp/tipp-submission/</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/tipp&amp;sa=D&amp;ust=1508540976081000&amp;usg=AFQjCNGKBx8YsLXmdqASnlqZXUdXkmJ4qg">https://sites.google.com/eng.ucsd.edu/datasets/tipp</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Mirarab and Warnow, FastSP</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">FastSP: Linear Time Calculation of Alignment Accuracy, Bioinformatics (2011)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c9 c19"><a class="c10" href="https://www.google.com/url?q=http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btr553&amp;sa=D&amp;ust=1508540976082000&amp;usg=AFQjCNEplLhSI3UYCNRVzvu1Beyb5KTaKg">10.1093/bioinformatics/btr553</a></span></p><p class="c2 c7"><span class="c3"></span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/software/fastsp&amp;sa=D&amp;ust=1508540976083000&amp;usg=AFQjCNE-SUJAwuS7Tu1k4zVz7pk4rgoKbg">www.cs.utexas.edu/~phylo/software/fastsp</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/fastsp&amp;sa=D&amp;ust=1508540976083000&amp;usg=AFQjCNHvpzpXRAgg4wgfjsOBLc2qQ-B35w">https://sites.google.com/eng.ucsd.edu/datasets/fastsp</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Liu et al., SATe-I</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">Rapid and Accurate Large-Scale Coestimation of Sequence Alignments and Phylogenetic Trees, Science (2009)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3">10.1126/science.1171243 </span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/sate/public/sate_journal.html&amp;sa=D&amp;ust=1508540976084000&amp;usg=AFQjCNFbMgFCE4ymM4S8lPtI0kx105KU3w">http://www.cs.utexas.edu/~phylo/sate/public/sate_journal.html</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/sate-i&amp;sa=D&amp;ust=1508540976084000&amp;usg=AFQjCNF-UG0t8TV6yMnp_dhQa23IgTZOtQ">https://sites.google.com/eng.ucsd.edu/datasets/sate-i</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Swenson et al.</span></p><p class="c2"><span class="c3">SuperFine</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">SuperFine: fast and accurate supertree estimation.</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3">10.1093/sysbio/syr092</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/superfine/&amp;sa=D&amp;ust=1508540976085000&amp;usg=AFQjCNEZQPr_T5GjvFZv0zMxS8GdbWB5sg">http://www.cs.utexas.edu/users/phylo/software/superfine/</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine&amp;sa=D&amp;ust=1508540976086000&amp;usg=AFQjCNEYwCXWzbkSeuMBzjDfzU_hlQeOqA">https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Nelesen et al., DACTAL</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">DACTAL: divide-and-conquer trees (almost) without alignments, Bioinformatics (2012)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3">10.1093/bioinformatics/bts218</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/dactal/&amp;sa=D&amp;ust=1508540976087000&amp;usg=AFQjCNGj_mL0xsZ_ph72k9xIwUt35I-rYQ">http://www.cs.utexas.edu/users/phylo/software/dactal/</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine&amp;sa=D&amp;ust=1508540976087000&amp;usg=AFQjCNGwfJws2afXHRPoysbc3eq-krzjzQ">https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Swenson et al., SMIDGEN</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">A simulation study comparing supertree and combined analysis methods using SMIDGen, Algorithms for Molecular Biology (2010)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3">https://doi.org/10.1186/1748-7188-5-8</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/mswenson/pubs/&amp;sa=D&amp;ust=1508540976088000&amp;usg=AFQjCNEFjQjXGsDbekiQ66guTHLDsIUaGA">http://www.cs.utexas.edu/users/mswenson/pubs/</a></span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine&amp;sa=D&amp;ust=1508540976089000&amp;usg=AFQjCNFsi0fcmne6eMSVNhhPM90YZhHWzw">https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine</a></span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Liu and Warnow (BeeTLe)</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">Treelength Optimization for Phylogeny Estimation, PLOS One (2012)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c22 c12">doi:10.1371/journal.pone.0033104</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/datasets/treelength/&amp;sa=D&amp;ust=1508540976091000&amp;usg=AFQjCNGG6UgjAw3sT8nU_tFOS4VKhWSiLA">http://www.cs.utexas.edu/users/phylo/datasets/treelength/</a></span><span class="c3">&nbsp;</span></p><p class="c2"><span class="c3">(data)</span></p><p class="c2 c7"><span class="c3"></span></p><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/beetle/beetle-may-17-2013.tar.bz2&amp;sa=D&amp;ust=1508540976092000&amp;usg=AFQjCNEN8YTgI1iDrotTEtXOfsj7pJ9VAQ">http://www.cs.utexas.edu/users/phylo/software/beetle/beetle-may-17-2013.tar.bz2</a></span><span class="c3">&nbsp;</span></p><p class="c2"><span class="c3">(software)</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine&amp;sa=D&amp;ust=1508540976093000&amp;usg=AFQjCNGgf2U5Cav7n-BhEBawWd6MmJeCaA">https://sites.google.com/eng.ucsd.edu/datasets/dactalsuperfine</a></span></p><p class="c2 c7"><span class="c3"></span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Liu et al.</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c3">RAxML and FastTree: Comparing Two Methods for Large-Scale Maximum Likelihood Phylogeny Estimation, PLOS One (2012)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c3 c12">10.1371/journal.pone.0027731</span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/sate/public/sate_journal.html&amp;sa=D&amp;ust=1508540976094000&amp;usg=AFQjCNExLwG0GLMtA7vujuKCkpPDT007yg">http://www.cs.utexas.edu/~phylo/sate/public/sate_journal.html</a></span><span class="c3">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/sate-i&amp;sa=D&amp;ust=1508540976095000&amp;usg=AFQjCNE5oVN5t-byPEO83HChz7sNd1l9GQ">https://sites.google.com/eng.ucsd.edu/datasets/sate-i</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><p class="c2"><span class="c3">Yang and Warnow</span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c22">Fast and accurate methods for phylogenomic analyses, BMC Bioinformatics (2011)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c22 c12 c26"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.1093/bioinformatics/btt394&amp;sa=D&amp;ust=1508540976096000&amp;usg=AFQjCNHYhJ3bhNh67BRakSrLs808y4yQrg">10.1093/bioinformatics/btt394</a></span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c13"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/datasets/ILS/&amp;sa=D&amp;ust=1508540976096000&amp;usg=AFQjCNEOq1cS5LlGko_oBgjcOmTre4EV4A">http://www.cs.utexas.edu/users/phylo/datasets/ILS/</a></span><span class="c9">&nbsp;</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/ils-small&amp;sa=D&amp;ust=1508540976097000&amp;usg=AFQjCNFE7m5t4vQ7ACKCxphVYnWuwh5Mcw">https://sites.google.com/eng.ucsd.edu/datasets/ils-small</a></span><span class="c3">&nbsp;</span></p></td></tr><tr class="c4"><td class="c16" colspan="1" rowspan="1"><h1 class="c21" id="h.oddejvxfkokm"><span class="c22 c29">Bayzid, Hunt, and Warnow </span></h1><p class="c2 c7"><span class="c3"></span></p></td><td class="c1" colspan="1" rowspan="1"><p class="c2"><span class="c22">Disk covering methods improve phylogenomc analyses, BMC Genomics (2014)</span></p></td><td class="c5" colspan="1" rowspan="1"><p class="c2"><span class="c22 c12 c27"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.1186/1471-2164-15-S6-S7&amp;sa=D&amp;ust=1508540976098000&amp;usg=AFQjCNF7hKuhu38RUNj4DrJRpoQrSYTdfA">10.1186/1471-2164-15-S6-S7</a></span></p></td><td class="c6" colspan="1" rowspan="1"><p class="c2"><span class="c9 c12"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/users/phylo/software/dcm-protocol/&amp;sa=D&amp;ust=1508540976099000&amp;usg=AFQjCNEDtribVivnacPPL2uPaevYcvi5hA">http://www.cs.utexas.edu/users/phylo/software/dcm-protocol/</a></span><span class="c22 c12">&nbsp;</span><span class="c3 c12">(scripts)</span></p><p class="c2 c7"><span class="c3 c12"></span></p><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=http://www.cs.utexas.edu/~phylo/datasets/astral/&amp;sa=D&amp;ust=1508540976099000&amp;usg=AFQjCNFOBb8qwYg7WCjdyV5RLhFCINB1tQ">http://www.cs.utexas.edu/~phylo/datasets/astral/</a></span><span class="c22">&nbsp;</span><span class="c22">(datasets)</span></p></td><td class="c15" colspan="1" rowspan="1"><p class="c2"><span class="c9"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/ils-small&amp;sa=D&amp;ust=1508540976100000&amp;usg=AFQjCNFtBFhmwMnYall3wa66Z_ALrne0Sw">https://sites.google.com/eng.ucsd.edu/datasets/ils-small</a></span><span class="c3">&nbsp;<br>(scripts)</span></p><p class="c2 c7"><span class="c3"></span></p><p class="c2"><span class="c9 c12"><a class="c10" href="https://www.google.com/url?q=https://sites.google.com/eng.ucsd.edu/datasets/astral&amp;sa=D&amp;ust=1508540976100000&amp;usg=AFQjCNET0QFRWZ8Dgrs1Pzy_VgR6yR8Zgw">https://sites.google.com/eng.ucsd.edu/datasets/astral</a></span><span class="c9 c12">&nbsp;</span><span class="c3">&nbsp;(datasets)</span></p></td></tr></tbody></table>
